github_review_helper_suite_test.go

```diff
@@ -0,0 +1,17 @@
+package main_test
+
+import (
+	"io/ioutil"
+	"log"
+
+	. "github.com/onsi/ginkgo"
```

Comment:
Unfortunately the standard go tools don't make that possible. Luckily most gophers (which is what people writing go call themselves :)) are aware of that and try to keep their packages backwards compatible. 

Most mission critical software written in go uses vendoring. Unfortunately, there's no well established pattern for that either. There are some early tools for vendoring in the official toolset, but they're still quite fresh and limited.

All in all, I'd like to avoid vendoring/dependency versioning until it starts causing us problems.